packages/patternfly-4/react-core/src/components/Chip/examples/SimpleChip.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { Chip } from '@patternfly/react-core';
+
+class SimpleChip extends React.Component {
+  state = {
+    show: ['chip_one', 'chip_two']
+  };
+  deleteItem = id => {
+    const { show } = this.state;
+    const ix = show.indexOf(id);
+    if (ix < 0) {
+      return;
+    }
+    this.setState(() => ({
+      show: [...show.slice(0, ix), ...show.slice(ix + 1, show.length)]
+    }));
+  };
+  render() {
+    return (
+      <React.Fragment>
+        {this.state.show.includes('chip_one') && (
```

Comment:
The component looks great - nice work. Just a few things - can we leverage destructuring here for cleaning up the render method of state references?

Reply:
Yeah sure!